docs/using-guidelines.asciidoc

```diff
@@ -1,11 +1,25 @@
-//[[ecs-guidelines]]
-== Guidelines and Best Practices
+[[ecs-guidelines]]
+=== Guidelines and Best Practices
```

Comment:
This asciidoc seems to overlap with the converting doc.  This might be OK, but the information between the two needs to be reconciled.